src/main/java/net/fabricmc/loader/impl/discovery/RuntimeModRemapper.java

```diff
@@ -234,6 +258,7 @@ private static class RemapInfo {
 		boolean inputIsTemp;
 		OutputConsumerPath outputConsumerPath;
 		String accessWidenerPath;
+		byte[] sourceAccessWidener;
 		byte[] accessWidener;
```

Comment:
Nit: I dont think there is a need for `sourceAccessWidener` you can just set `accessWidener` and then replace it after its been remapped.